.gitignore

```diff
@@ -3,6 +3,8 @@
 .venv
 .mypy_cache
 .idea
+/backend/apicache/
+/backend/package-lock.json
```

Comment:
hmm, why is this one needed? I believe the package-lock.json should only be in the `web` dir